### DISCUSSION TITLE
Preview 13?

### DISCUSSION THREAD

**Discussion by nogginbox** [dotnet/maui#4685] [Q&A] [answered]
Is preview 13 going to be released soon? There are some bug fixes that I'd really like to get soon.

**Comment by yyliveyy** [comment]
The same question.

**Comment by nogginbox** [comment, marked as answer]
I'm going to answer my own question and say whoop WHOOP, it's here.

Here is the [blog post on it's release](https://devblogs.microsoft.com/dotnet/announcing-net-maui-preview-13/).

I'm just downloading the latest VS 2022 Preview. Fingers crossed it fixes some of the issues I've been having.
